manuscript/00-foreword/00-introduction.md

```diff
@@ -1,13 +1,13 @@
 # Introduction
 
-The central theme of this book is GraphQL in modern applications. While GraphQL is only a specification that can be used in many programming languages, this book focuses on GraphQL in JavaScript. Facebook--the company behind the GraphQL--didn't release GraphQL's reference implementation in JavaScript, but it remains one of the more compelling languages for mordern client-server architecture. Using Facebook's specification as their blueprint, other programming languages like Python and Java followed. 
+Modern web applications are often bloated and complex when they're implemented as full-blown client-server architectures. There are usually a fontend and backend applications, a database, and an interface that lets both ends communicate. In my other books I teach about React.js for frontend applications, Node.js for backend applications with databases, and GraphQL or REST for the communication between them. Doing all of this yourself can be a lengthy endeavor, though, so if you're looking for a stop-gap solution, Firebase offers features like database connectivity and authentication by default.
```

Comment:
"usually a fontend and backend applications" -> "usually a fontend and backend application"